packages/utilities/psammead-styles/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-styles",
-  "version": "7.0.3",
+  "version": "7.0.4",
```

```suggestion
  "version": "7.1.0",
```
As [with the spacing PR](https://github.com/bbc/psammead/pull/4244#discussion_r559996973) I'd say this should be a minor version bump; I appreciate btw others haven't bumped a minor version in similar past changes.